README.rst

```diff
@@ -28,133 +28,43 @@ Nengo: Large-scale brain modelling in Python
 Installation
 ============
 
-To install Nengo, use::
+To install Nengo::
```

Comment:
Do we also want information on how to install the dev version? If people are reading this, they're either on the Github site or have downloaded the source code, so it might be good to briefly tell them how to install from source.